tests/test_surveys_submitted.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from django.urls import reverse
+
+
+@pytest.mark.django_db
+def test_survey_submitted_list(user, survey, client, form_entry):
```

Comment:
What do you think about making this file `tests/test_views.py`? `tests/test_surveys_submitted.py` makes sense if we anticipate we'll have a lot of tests for submitted surveys in particular, otherwise this test might fit better in a category that encompasses "all tests checking basic attributes of views".

Reply:
love it